src/libraries/index.tsx

```diff
@@ -107,13 +107,13 @@ export const libraries = [
   pacerProject,
   storeProject,
   rangerProject,
-  // optimisticProject,
+  dbProject,
   configProject,
 ] satisfies Library[]
 
 export const librariesByGroup = {
   app: [startProject, routerProject],
-  state: [queryProject, storeProject, pacerProject],
+  state: [queryProject, dbProject, storeProject, pacerProject],
```

Comment:
we'll combine `app` and `state` sections as discussed in discord. Keep order that they are in with query at 3, db at 4 etc.